docs/cron-workflows.md

```diff
@@ -103,6 +104,26 @@ For example, with timezone set at `America/Los_Angeles`, we have daylight saving
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+### Stopping a `CronWorkflow`
+
+> v3.6 and after
+
+It is possible to stop the execution of new workflows from a `CronWorkflow` based on the number of previous successes or failures. The spec field `stopStrategy` accepts a `condition` that stops triggering new workflows when it evaluates to true.
+
+Keywords allowed for stopping conditions are `failed` and `succeeded`:
+
+- `failed` - counts the number of workflows that failed to submit or completed in phase `Error` or `Failed`
+- `succeeded` - counts the number of workflows that completed in phase `Succeeded`
+
+For example, if you want a workflow to be executed only once you can define the spec as:
+
+```yaml
+stopStrategy:
+  condition: succeeded >= 1
+```
```

Comment:
Can we add the example for failed as well?

Reply:
done thanks